src/gui/mzroll/mainwindow.cpp

```diff
@@ -3249,42 +3239,30 @@ QWidget* MainWindowWidgetAction::createWidget(QWidget *parent) {
 
 		QToolButton *btnNext = new QToolButton(parent);
 		btnNext->setIcon(QIcon(rsrcPath + "/next.png"));
-		btnNext->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
 		btnNext->setToolTip(tr("History Forward (Ctrl+Right)"));
 		btnNext->setShortcut(tr("Ctrl+Right"));
 		connect(btnNext, SIGNAL(clicked()), mw, SLOT(historyNext()));		
 		return btnNext;
 
 	}
-	else if (btnName == "btnPDF") {
-
-		QToolButton *btnPDF = new QToolButton(parent);
-		btnPDF->setIcon(QIcon(rsrcPath + "/exportpdf.png"));
-		btnPDF->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPDF->setToolTip(tr("Save EIC Image to PDF file"));
-		connect(btnPDF, SIGNAL(clicked()), mw, SLOT(exportPDF()));
-		return btnPDF;
-
-	}
-	else if (btnName == "btnPNG") {
-
-		QToolButton *btnPNG = new QToolButton(parent);
-		btnPNG->setIcon(QIcon(rsrcPath + "/copyPNG.png"));
-		btnPNG->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPNG->setToolTip(tr("Copy EIC Image to Clipboard"));
-		connect(btnPNG, SIGNAL(clicked()), mw, SLOT(exportSVG()));
-		return btnPNG;
-
-	}
-	else if (btnName == "btnPrint") {
-
-		QToolButton *btnPrint = new QToolButton(parent);
-		btnPrint->setIcon(QIcon(rsrcPath + "/fileprint.png"));
-		btnPrint->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPrint->setToolTip(tr("Print EIC (Ctr+P)"));
-		btnPrint->setShortcut(tr("Ctrl+P"));
```

Comment:
You forgot to add this. The shortcut should still remain

Reply:
Sorry about that. Fixed.